lib/plugins/deploy/README.md

```diff
@@ -1,12 +1,14 @@
 # Deploy
 
 ```
-serverless deploy
+serverless deploy [function]
 ```
```

Comment:
Yes. It's a nested command in the `deploy` plugin (the `[]` should show that this is optional).